### PR TITLE
Updated memakerwrap.py

### PR DESCRIPTION
**Pull request by microSharjeel**

### CODE REVIEW

File: software/python/memakerwrap.py [hunk 1]
```diff
@@ -169,111 +169,206 @@ def instWires (type, async, be) :
...
-    if tech == "LD130":
+    if (tech == "LD130"):
         if type == "SZ": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"CM"+str(mux)+" regf"
         elif type == "SJ": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"CM"+str(mux)+" ram"
         elif type == "SH": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"BM"+str(mux)+" ram"
         elif type == "SP": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"BM"+str(mux)+"A rom"
-    
+    if (tech=="skywater130"):
+ 	
+ 	if type == "SZ": print "   "+tech+type+"_"+str(2**words)+"X"+str(bits*bytes)+"_"+str(bits)+" regf"
```
Comment (JDLopes) [inline software/python/memakerwrap.py:182]: Is an underscore missing between tech and type? Please, check ifs indentation.

File: software/python/memakerwrap.py [hunk 2]
```diff
@@ -389,6 +484,48 @@ def main () :
                 mems.append([words, bits, mux])
         else:
             sys.exit("Unsupported memory type")
+    elif sys.argv[1] == "skywater130":
```
Comment (JDLopes) [inline software/python/memakerwrap.py:487]: The tech is skywater130 or just sky130? In OpenRAM examples, sky130 is used.

File: software/python/memakerwrap.py [hunk 3]
```diff
@@ -169,111 +169,206 @@ def instWires (type, async, be) :
...
 def instMemory (tech, type, words, bits, bytes, mux):
     # memory simulation modute name
-    if tech == "LD130":
+    if (tech == "LD130"):
         if type == "SZ": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"CM"+str(mux)+" regf"
         elif type == "SJ": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"CM"+str(mux)+" ram"
         elif type == "SH": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"X"+str(bytes)+"BM"+str(mux)+" ram"
         elif type == "SP": print "   "+type+tech+"_"+str(2**words)+"X"+str(bits)+"BM"+str(mux)+"A rom"
-    
+    if (tech=="skywater130"):
```
Comment (JDLopes) [inline software/python/memakerwrap.py:180]: Please add spaces between the equal signs and the rest. It makes it easier to read the code.

File: software/python/memakerwrap.py [hunk 4]
```diff
@@ -389,6 +484,48 @@ def main () :
                 mems.append([words, bits, mux])
         else:
             sys.exit("Unsupported memory type")
+    elif sys.argv[1] == "skywater130":
+        tech = "skywater130"
+        moduleName = sys.argv[2]
+        if sys.argv[3] == "SZ":
```
Comment (JDLopes) [inline software/python/memakerwrap.py:490]: These types (SZ, SJ, SH and SP) came from Memaker tool. They don't exist in OpenRAM. I suggest you use spram, dpram, sprom, spregf, etc, as types. Then, for coding reuse, you associate them with the ones used for Memaker: register file (SZ), or dual-port RAM (SJ), or single-port RAM (SH), or single-port ROM (SP), by simply setting the type variable to right one.

**Review by JDLopes — CHANGES_REQUESTED** [review]
Pay attention to indentation for Python scripts. Use your text editor (or IDE if you use one) with the right settings, so tabs are not mixed up with spaces. It seems to me that you have one or two lines badly indented.

Please, check the code lines indentation.

File: software/python/memakerwrap.py (hunk 4 above)
Comment (microSharjeel) [reply software/python/memakerwrap.py:490]: ok u mean that I should use proper names that associate with OpenRAM?

File: software/python/memakerwrap.py (hunk 1 above)
Comment (microSharjeel) [reply software/python/memakerwrap.py:182]: it is ok

File: software/python/memakerwrap.py (hunk 2 above)
Comment (microSharjeel) [reply software/python/memakerwrap.py:487]: it is skywater130

File: software/python/memakerwrap.py (hunk 3 above)
Comment (microSharjeel) [reply software/python/memakerwrap.py:180]: done!

File: software/python/memakerwrap.py (hunk 4 above)
Comment (JDLopes) [reply software/python/memakerwrap.py:490]: I mean that in OpenRAM examples, the names used for type are different from the ones used in Memaker. As I suggested earlier, you can use spram, dpram, sprom, etc, as type in OpenRAM. They are more easy to read, and when you assign type variable in main() function, while extracting the arguments, you should use the ones that Memaker uses (SZ, SJ, SH and SP).

**Review by JDLopes — CHANGES_REQUESTED** [review]
Please, check the comments

**Pull request closed without merging**